spec/integration/associations_spec.rb

```diff
@@ -286,6 +286,35 @@ class Publisher < ActiveFedora::Base
       end
     end
 
+    describe "when fetching an existing object" do
+      before do
+        class Dog < ActiveFedora::Base; end
+        class BigDog < Dog; end
+        class Cat < ActiveFedora::Base; end
+        @dog = Dog.create()
+        @bigDog = BigDog.create()
```

Comment:
:bomb: the empty parens, and prefer snake_case over camelCase for variable names. :snake: > :camel: